features/collections/collectible.feature

```diff
@@ -112,3 +112,70 @@ Feature: Collectible items
     Then I should see "Bookmarks (0)"
       And I should not see "Tundra penguins"
 
+  Scenario: Bookmarks of deleted items are included on the collection's Manage
```

Comment:
Only a suggestion so don't worry if you don't have time, but these scenarios could be extracted into a separate "Deleted Items in Collections" feature with the Background:
```
   Given I have a collection "Various Penguins"
      And I am logged in as a random user
```
(no big deal though)